### ISSUE TITLE
softwarekeep.com: ads

### ISSUE THREAD

**Issue by saveminecraft1** [uBlockOrigin/uAssets#19230]
### Prerequisites

- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Use this [button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out.)
- [X] I forced an update of my filter lists. (Click the ["Purge all caches"](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#purge-all-caches) button while holding the 'Shift' key, then click the ["Update now"](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#update-now) button.)
- [X] I did not remove any of the [default filter lists](https://user-images.githubusercontent.com/124740436/235392297-c4fc6290-73a6-4a70-96a9-3f6a7683f21d.png), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable additional filter lists, or I have verified that the issue still occurs without enabling additional filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin (uBO) along with other content blocker extensions.
- [X] I have verified that the web browser's built-in blocker or [DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking) (standalone or through a VPN) is not causing the issue.
- [ ] I did not answer truthfully to **ALL** the above checkpoints.

### URL address of the web page

`https://softwarekeep.com/help-center/use-function-keys-without-pressing-fn-key-on-windows-10`

### Category

ads

### Description

Ads still shown.

### Other extensions used

None

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

![image](https://github.com/uBlockOrigin/uAssets/assets/77214848/fc228522-2344-4369-a142-e9bf6a6a20e6)

</details>

### Configuration

<details>

```yaml
uBlock Origin: 1.51.0
Firefox: 115
filterset (summary):
 network: 97753
 cosmetic: 39557
 scriptlet: 18126
 html: 1013
listset (total-discarded, last-updated):
 default:
  user-filters: 0-0, never
  ublock-filters: 35011-142, 4m
  ublock-badware: 7623-2, 4m
  ublock-privacy: 482-0, 4m
  ublock-unbreak: 2153-1, 4m
  ublock-quick-fixes: 308-0, 4m
  easylist: 69070-667, 4m
  easyprivacy: 33365-761, 4m
  urlhaus-1: 6446-0, 4m
  plowe-0: 3717-1, 4m
filterset (user): [empty]
trustedset:
 removed:
  about-scheme
  chrome-extension-scheme
  chrome-scheme
  edge-scheme
  moz-extension-scheme
  opera-scheme
  vivaldi-scheme
  wyciwyg-scheme
userSettings: [none]
hiddenSettings: [none]
supportStats:
 allReadyAfter: 829 ms
 maxAssetCacheWait: 239 ms
popupPanel:
 blocked: 10
 network:
  adroll.com: 1
  facebook.net: 1
  googlesyndication.com: 1
  googletagmanager.com: 3
  klaviyo.com: 2
  m2.ai: 1
  symantec.com: 1
```
</details>

**Comment by saveminecraft1** [comment]
Yes I saw the other post but the comment said they where using `AdNauseam` and its not fixed anyways.

**Comment by stephenhawk8054** [comment]
Can you point in the screenshot what are the ads you want to imply?

**Comment by saveminecraft1** [comment]
Ads on the side and top
